community/kernel/src/main/java/org/neo4j/diagnostics/DiagnosticsReporterProgressInteractions.java

```diff
@@ -48,6 +48,14 @@
      */
     void error( String msg, Throwable throwable );
 
+    /**
+     * A check whether to ignore the free available disk space when generating the archive.
+     *
+     * @param message with information about the disk space.
+     * @return true if the execution should continue even if risking to fill up the disk.
+     */
+    boolean shouldIgnorePotentialFullDisk( String message );
```

Comment:
it feels a bit wrong that its a progress reporter who decide should or shouldn't we ignore potential OOD problems.